ethereum-consensus/src/state_transition/error.rs

```diff
@@ -55,6 +55,8 @@ pub enum Error {
     UnknownPreset(String),
     #[error(transparent)]
     ExecutionEngine(#[from] ExecutionEngineError),
+    #[error(transparent)]
+    PolynomialCommitment(#[from] crate::deneb::polynomial_commitments::Error),
```

Comment:
easier to read if you do the import at the top and just bind some local symbol here

```rust
use crate::deneb::polynomial_commitments::Error as PolynomialCommitmentError;
```

and then just 
```suggestion
    PolynomialCommitment(#[from] PolynomialCommitmentError),
```